docs/misc/deployment/docker-compose-full-example.md

```diff
@@ -0,0 +1,91 @@
+# Full Tutorial for Server Setup using Docker Compose
+
+This is a reference guide/tutorial on how to set up a Minecraft server using this project and docker-compose. Some parts of this tutorial will make mention of operating system-dependent methods and may or may not provide the specific workflow for your OS (e.g. Windows, Mac, Linux, etc.). Please access the many readily available online resources on how to perform these steps.
+
+## 1. Set RCON_PASSWORD passphrase (if you plan to use RCON)
+
+The very first and most important thing to mention is that creating a Minecraft server comes with security risks, and by creating a Minecraft server using any method at all, you accept these risks. *Understanding what these risks are and how to minimize your risk is of the utmost importance.*
+
+RCON is enabled by default to perform accessory functions of this project. Without an RCON password set, it will be set to a randomized value each time you start the server. If you plan to use an RCON password (which might be sensible if you plan to use RCON), then you should set this first before anything else for convenience and because it will improve your server setup workflow. **DO NOT reuse a passphrase that you use for anything else, as this poses a significant security risk.** It is better is to memorize your passphrase rather than putting it anywhere, but if you do, guard it like you would any other passphrase by putting it in a safe place - *don't place it directly in your server folder*.
+
+## 2. Enable Virtualization.
+
+The second step is to enable virtualization (this varies by System). First, check if you already have virtualization enabled - please follow the appropriate steps according to your operating system. Specific steps are out of scope for the purposes of this tutorial, but if you find that you do not have virtualization enabled, you will need to enable it in this step for Docker to work.
+
+## 3. Download Docker Desktop.
+
+You may navigate to [this page to download Docker Desktop](https://www.docker.com/products/docker-desktop/). It is required to run this project.
+
+## 4. Create a New Server Directory
+
+Create a new folder and name it something recognizable (such as "minecraft-server", for example). You may either use the command line, or your system's file manager GUI.
+
+## 5. Create and Populate docker-compose.yml File in the Server Directory
+
+The most important part of the docker-compose.yml creation step is to make sure that your docker-compose.yml file is properly formatted. One potential example docker-compose.yml file is as follows (be sure to delete any irrelevant lines to your application, or add any that you may need before copying and pasting the file contents). Be sure that the indentation is correct for every part of the file. 
+
+!!!note
```

Comment:
There's a mistake here, missing a space! Didn't even notice it while I was pushing. Just letting you know that it is there.

Thank you once again for your attention towards this matter and for considering allowing me to contribute to this wonderful project.